plugin/trino-hive/src/main/java/io/trino/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -539,6 +544,7 @@ private void invokeNoMoreSplitsIfNecessary()
             }
         }
         else {
+            acidInfoBuilder.setOrcAcidVersionValidated(true); // no ACID; no further validation needed
```

Comment:
why would we need to set this for "no ACID" case?

Reply:
https://github.com/trinodb/trino/pull/7579#issuecomment-818800106

Reply:
let's have an followup issue

Reply:
https://github.com/trinodb/trino/issues/7603
Added `//TODO`